internal/api/shared/schemas/query_disjunction_generator.cue

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gen
```

Comment:
shouldn't be `package base` ?

Reply:
the name doesn't matter actually, and I put a different one to highlight the fact it's not used for the same purpose
but I will change for a full "generator" instead

Reply:
maybe add a comment to explain this part. It's a bit counter intuitive it's not the same name, but makes sense once you explained.